lib/menu.rb

```diff
@@ -0,0 +1,57 @@
+require 'csv'
+require_relative 'dish'
+
+class Menu
+
+  OUTPUT_WIDTH = 60
+
+  attr_reader :dishes
+
+  def initialize(filename = 'menu.csv')
+    @filename = filename
+    load_menu
+  end
+
+  def load_menu(filename = @filename)
+    file_not_exist?(filename)
+    load_data_into_hash(filename)
+  end
+
+  def print_menu(output_width = OUTPUT_WIDTH)
+    print_header(output_width)
+    print_body(output_width)
+  end
+
+  private
+
+  def load_data_into_hash(filename)
+    @dishes = Hash.new
+    CSV.foreach(filename) do |row|
+      dish = Dish.new
+      dish.name, dish.ingredients, dish.price, dish.type  = row[0], row[1], row[2].to_i, row[3]
```

Comment:
A little bit confused how to use keyword arguments in this case. Two options that work for me are 
(right now I am creating Dish as Dish = Struct.new(:name, :ingredients, :price, :type))
(1) dish = Dish.new(row[0], row[1], row[2].to_i, row[3]) - don't like this one as it makes it unclear which row corresponds to what
and (2)  dish = Dish.new
      dish.name, dish.ingredients, dish.price, dish.type  = row[0], row[1], row[2].to_i, row[3]